package.json

```diff
@@ -28,7 +28,7 @@
   "homepage": "https://github.com/uc-cdis/guppy#readme",
   "dependencies": {
     "@elastic/elasticsearch": "~7.13.0",
-    "@gen3/ui-component": "^0.11.0",
+    "@gen3/ui-component": "uc-cdis/gen3-ui-component#fix/dep",
```

Comment:
I think we need to release the new version of `gen3-ui-component` instead of using the one in feature branch

Reply:
yes, please review the ui-component PR first